packages/flame/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
  - Introduce `scale` on `PositionComponent`
  - Add `ScaleEffect` that works on `scale` instead of `size`
  - Add class `NotifyingVector2`
+ - Fix `scale` in `PositionComponent`
```

Comment:
Just a general question, but this is a fix for a change that wasn't released yet, do we need to have a CHANGELOG entry then? As it doesn't "change" anything for the public facing user?

Reply:
I agree, no need for this entry

Reply:
True, I'll remove it. Then it is not public facing that I broke it either. ;)